planet.js

```diff
@@ -0,0 +1,57 @@
+let nameH1;
+let filmsDiv;
+let planetDiv;
+let charactersDiv;
+const baseUrl = `https://swapi2.azurewebsites.net/api`;
+
+// Runs on page load
+addEventListener('DOMContentLoaded', () => {
+  nameH1 = document.querySelector('h1#name');
+  charactersUl = document.querySelector('#characters>ul');
+  filmsUl = document.querySelector('#films>ul');
+
+  const sp = new URLSearchParams(window.location.search)
+  const id = sp.get('id')
+  getPlanet(id)
+});
+
+async function getPlanet(id) {
+  let planet;
+  try {
+    planet = await fetchPlanet(id)
+    planet.films = await fetchFilms(planet)
```

Comment:
After line 22, need to add planet.characters fetch

Reply:
Changes Done